packages/trader/src/Modules/Contract/Components/AccumulatorsStats/__tests__/ticks-history-counter.spec.tsx

```diff
@@ -2,8 +2,10 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import TicksHistoryCounter from '../ticks-history-counter';
 
+type TTicksHistoryCounter = React.ComponentProps<typeof TicksHistoryCounter>;
+
 describe('TicksHistoryCounter', () => {
-    let mock_props;
+    let mock_props: TTicksHistoryCounter;
```

Comment:
```suggestion
    let mock_props: React.ComponentProps<typeof TicksHistoryCounter>;
```
@henry-deriv it's used just once, so we can use it straight without declaring a variable🙂